autoscheduler/frontend/src/webpack.config.js

```diff
@@ -35,7 +35,10 @@ module.exports = {
           {
             loader: 'typings-for-css-modules-loader',
             options: {
-              modules: true,
+              modules: {
+                mode: 'local',
+                localIdentName: '[path][name]__local',
```

Comment:
This line should theoretically give us human-readable class names for CSS modules, but it doesn't seem to be working

Reply:
Might be related to the version of css-loader we're using (1.0.1, rather than 3.x)

Reply:
Fixed. If you inspected element before, the CSS classes would've been long Base64 strings. Now they have the format `[component name]--[last 5 chars of base64 hash]`